chaincode/tsconfig.lib.json

```
@@ -8,7 +8,7 @@
     "composite": true,
     "types": ["node"]
   },
-  "include": ["src/**/*", "package.json"],
+  "include": ["src/**/*", "package.json", "../chain-api/src/types/DexUserPositions.ts"],
```

```suggestion
  "include": ["src/**/*", "package.json"],
```

This broke the CI/CD build, as we see here: https://github.com/GalaChain/sdk/actions/runs/13461741671/job/37618510302?pr=508

Please work with the existing monorepo setup / nx, do not try to include relative paths to other modules.